src/main/presenter/mcpPresenter/inMemoryServers/appleServer.ts

```
@@ -0,0 +1,1770 @@
+                text: `Found ${searchResults.length} events matching "${parsedArgs.searchText}":\n\n${searchResults.map(event => 
+                  `• ${event.title} (${event.startDate ? new Date(event.startDate).toLocaleDateString() : 'No date'})`
+                ).join('\n')}`
+              }
+            ]
+          }
+
+        case 'list':
+          const events = await CalendarUtils.getEvents(
+            parsedArgs.limit,
+            parsedArgs.fromDate,
+            parsedArgs.toDate
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)